packages/sns/src/converters/governance.converters.spec.ts

```diff
@@ -196,39 +224,32 @@ describe("governance converters", () => {
           new_controllers: [mockPrincipal],
         },
       };
-      const expectedAction: Action = {
-        DeregisterDappCanisters: {
-          canister_ids: [mockPrincipal],
-          new_controllers: [mockPrincipal],
-        },
-      };
-      expect(toActionOptional(action)).toEqual(expectedAction);
+      expect(toActionOptional(action)).toEqual(action);
     });
 
     it("converts Unspecified action", () => {
       const action: ActionCandid = {
         Unspecified: {},
       };
-      const expectedAction: Action = {
-        Unspecified: {},
-      };
-      expect(toActionOptional(action)).toEqual(expectedAction);
+      expect(toActionOptional(action)).toEqual(action);
     });
 
     it("converts ManageSnsMetadata action", () => {
+      const url = "https://example.com";
+      const name = "New name";
       const action: ActionCandid = {
         ManageSnsMetadata: {
-          url: ["https://example.com"],
+          url: [url],
           logo: [],
-          name: ["New name"],
+          name: [name],
           description: [],
```

Comment:
No description?

Reply:
I also want to test that it returns `undefined` in some fields.

Reply:
logo is already undefined, right?